content/applications/inventory_and_mrp/inventory/shipping/setup/delivery_method.rst

```diff
@@ -1,100 +1,215 @@
+weight, volume, price, or quantity is compared to a defined amount to calculate the
+:guilabel:`Delivery Cost`.
+
+Once finished, click either :guilabel:`Save & New` to add another rule, or :guilabel:`Save & Close`.
+
+.. example::
+   To charge customers $20 in shipping for orders with five or fewer products, set the
+   :guilabel:`Condition` to `Quantity <= 5.00`, and the :guilabel:`Delivery Cost` to `$20`.
+
+   .. image:: delivery_method/pricing-rule.png
+      :align: center
+      :alt: Display window to add a pricing rule. Set a condition and delivery cost.
+
+To restrict shipping to specific destinations on the eCommerce website, in the shipping method form,
+navigate to the :guilabel:`Destination Availability` tab and define the :guilabel:`Countries`,
+:guilabel:`States`, and :guilabel:`Zip Prefixes`. Leave these fields empty if all locations apply.
+
+Calculate delivery cost
+~~~~~~~~~~~~~~~~~~~~~~~
+
+Shipping cost is the :guilabel:`Delivery cost` specified in the rule that satisfies the
+:guilabel:`Condition`, plus any extra charges from the :guilabel:`Margin on rate` and
+:guilabel:`Additional margin`.
+
+.. math::
+   Total = Rule's~Delivery~Cost + (Margin~on~rate \times Rule's~Delivery~Cost) + Additional~margin
+
+.. example::
+   With the two following rules set up:
+   #. If the order contains five or fewer products, shipping is $20
+   #. If the order contains more than five products, shipping is $50.
+
+   :guilabel:`Margin on Rate` is `10%` and :guilabel:`Additional margin` is `$9.00`.
 
-On the delivery, check the **Carrier Information**. The carrier is the
-chosen delivery method.
+   .. image:: delivery_method/delivery-cost-example.png
+      :align: center
+      :alt: Show example of "Based on rules" shipping method with margins configured.
+
+   When the first rule is applied, the delivery cost is $31 (20 + (0.1 * 20) + 9). When the second
+   rule is applied, the delivery cost is $64 (50 + (0.1 * 50) + 9).
+
+.. _inventory/shipping/pickup:
+
+Pickup in store
+---------------
+
+To configure in-store pickup, select :guilabel:`Pickup in store` in the :guilabel:`Provider` field
+and specify the pickup location in :guilabel:`Warehouse`.
+
+To invoice the customer for the shipping cost to the pickup location, choose the :guilabel:`Get Rate
+and Create Shipment` option in the :guilabel:`Integration Level` field. Then, pick either the
+:guilabel:`Estimated cost` or :guilabel:`Real cost` radio options in the :guilabel:`Invoicing
+Policy` field to decide whether the added shipping charge on the sales order is the precise cost
+from the shipping carrier.
+
+.. seealso::
+   :ref:`Invoice cost of shipping <inventory/shipping/invoice>`
+
+.. _inventory/shipping/sales-order:
+
+Add shipping
+============
+
+To apply the configured shipping methods to calculate shipping cost, navigate to the desired sales
+order by going to :menuselection:`Sales app --> Orders --> Orders`.
```

Comment:
Would be more specific here— as written (without the screenshot), it's not obvious that the shipping method(s) added before shows up as a delivery product in the order lines.

```suggestion
Shipping methods can be added to sales orders in the form of delivery products, which appear as
individual line items. First, navigate to the desired sales order by going to :menuselection:`Sales
app --> Orders --> Orders`.
```